app/models/User.coffee

```diff
@@ -501,6 +501,7 @@ module.exports = class User extends CocoModel
   useDexecure: -> not (features?.chinaInfra ? false)
   useSocialSignOn: -> not (features?.chinaUx ? false)
   showGemsAndXp: -> features?.classroomItems ? false
+  showHeroAndInventoryModals: -> features?.classroomItems ? false
```

Comment:
Should we rename this to be more scoped, or update the implementation to match this broadly scoped name?
E.g. `showHeroAndInventoryModalsToStudents` or `showHeroAndInventoryModals: -> features?.classroomItems ? not me.isStudent()`